sbncode/Metadata/FileCatalogMetadataSBN.h

```diff
@@ -6,11 +6,18 @@
 //
 //           FCL parameters:
 //
-//           FCLName        - FCL file name.
-//           FCLVersion     - FCL file version.
-//           ProjectName    - Project name.
-//           ProjectStage   - Project stage.
-//           ProjectVersion - Project version.
+//           FCLName         - FCL file name.
+//           ProjectName     - Project name.
+//           ProjectStage    - Project stage.
+//           ProjectVersion  - Project version.
+//           ProjectSoftware - Project software information.
+//           ProductionName  - Production campaign name.
+//           ProductionType  - Production campaign type.
+//           Merge           - Merge flag.
```

Comment:
I found this pretty cryptic. Can you explain further (in this comment?) what the merge fcl parameter represents, and what the merge parameters in the metadata represent?

Reply:
At this moment parameters merge.merge and merge.merged are just dumb sam parameters that don't do anything.  The intended semantic meaning of parameter merge.merge is that it flags a file that wants to be merged.  Parameter merge.merged flags a file that has been merged.  In MicroBooNE, parameter merge.merge causes files to flow from batch jobs into a special merging FTS dropbox which stores unmerged files temporarily on disk.  Both parameters are used to bookkeep the subsequent merging process.